### PR TITLE
finish user's activity

### PR DESCRIPTION
**Pull request by vjnhat**

### CODE REVIEW

File: app/views/supervisor/users/show.html.erb [hunk 1]
```diff
@@ -9,3 +9,15 @@
...
+<div class="col-md-8">
+  <% if @user.activities.any? %>
+    <h3> Activities <%= @user.activities.count %></h3>
+    <% @user.activities.each do |activity| %>
+      <ol class="row">
+        <%= activity.action %>
+        <%= activity.find_name %>
+      </ol>
+    <% end %>
+
```
Comment (NguyenTanDuc) [inline app/views/supervisor/users/show.html.erb]: remove this line

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -9,3 +9,14 @@
     </section>
   </aside>
 </div>
+<div class="col-md-8">
+    <% if @user.activities.any? %>
```
Comment (NguyenTanDuc) [inline app/views/users/show.html.erb]: 2 tab

File: app/models/user_task.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class UserTask < ActiveRecord::Base
+  belongs_to :User
+  belongs_to :Task
+  before_create :log_action
```
Comment (NguyenTanDuc) [inline app/models/user_task.rb]: add new line

File: app/models/course_user.rb [hunk 4]
```diff
@@ -1,4 +1,21 @@
 class CourseUser < ActiveRecord::Base
   belongs_to :user
   belongs_to :course
+
+  after_create :create_user_subject_task
+  after_create :log_action
+
+  def create_user_subject_task
```
Comment (dieunb) [inline app/models/course_user.rb]: private method

File: app/models/user_subject.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+class UserSubject < ActiveRecord::Base
+  belongs_to :User
+  belongs_to :Subject
+  before_create :log_action
+  before_update :create_msg
+
+  def log_action
```
Comment (dieunb) [inline app/models/user_subject.rb]: private method

File: app/models/user_task.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
+class UserTask < ActiveRecord::Base
+  belongs_to :User
+  belongs_to :Task
+
+  before_create :log_action
+  before_update :create_msg
+
+  def log_action
```
Comment (dieunb) [inline app/models/user_task.rb]: private method

**Pull request closed without merging**